src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background-color: $white-color;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-text-color;
+    margin: 0;
+  }
+
+  &__code {
+    @include text-secondary;
+  }
+
+  &__reviews {
+    @include display-position;
+
+    flex-direction: row;
```

Comment:
The 'flex-direction' property is redundant here as it defaults to 'row' for flex containers. You can safely remove this line unless you have a specific reason for including it.